labs/accessing-optimizely-event-data/README.md

```diff
@@ -0,0 +1,140 @@
+# Accessing Optimizely Enriched Event Data
+
+Optimizely [Enriched Events Export](https://docs.developers.optimizely.com/optimizely-data/docs/enriched-events-export) gives you secure access to your Optimizely event data so you can analyze your experiment results with greater flexibility. The export includes a useful combination of events attributes:
+
+- Raw metadata (event names, user IDs, etc) that you pass to Optimizely, without additional processing
+- Enriched metadata that Optimizely adds such as experiment IDs, variation IDs, and session IDs
+
+For more information, see the Enriche Events [data specification](https://docs.developers.optimizely.com/optimizely-data/docs/enriched-events-data-specification).
+
+This Lab contains `oevents`, a simple CLI tool for loading enriched event data.
+
+![oevents demo](img/demo.gif)
+
+## Prerequisites: bash (v4+), date, jq, curl, and aws
+
+`oevents` is written is [bash](https://www.gnu.org/software/bash/) and requires version 4 or greater to run.  It also requires the following prerequisites:
+
+- [`date`](https://www.gnu.org/software/coreutils/manual/html_node/date-invocation.html) (included in OS X and most GNU/Linux distributions)
+- [`jq`](https://stedolan.github.io/jq/)
+- [`curl`](https://curl.haxx.se/)
+- the [Amazon AWS CLI](https://aws.amazon.com/cli/) (v2+) 
+
+## Installation
+
+`oevents` is a bash script. To use it, you can specify the script's path explicitly like this (assuming `oevents` is in your working directory):
+
+```sh
+$ ./oevents help
+```
+
+Or you can add it to a directory in your `PATH` environment variable.  You can do this temporarily by navigating to this tutorial's directory and adding it to your `PATH` variable:
```

Comment:
Maybe have "tutorial's directory" link to the github repo: https://github.com/optimizely/labs/blob/master/labs/accessing-optimizely-event-data

Since on the labs page it might not be clear.

Reply:
I removed this particular suggestion and added a separate "Downloading" section.  Included the link there.